src/Components/LoginBox/LoginBox.jsx

```diff
@@ -0,0 +1,97 @@
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import * as styles from "./LoginBox.module.css";
+
+export const LoginBox = ({ setAuth }) => {
+  //   const { login, testLogger } = useAuthContext();
+  const [userDetails, setUserDetails] = useState({ email: "", password: "" });
+  const [error, setError] = useState({ state: false, text: "" });
+  const [type, setType] = useState(true);
+  const [trigger, setTrigger] = useState(true);
+  //   useDocTitle("Login | StormKeep");
+
+  const validation = /^(?=.*\d)(?=.*[a-z]).{5,10}$/;
+  const emailValidation =
+    /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+
+  useEffect(() => {
+    const timeout = setTimeout(() => {
+      setError({
+        ...error,
+        state: false,
+      });
+    }, 3000);
+    return () => clearTimeout(timeout);
+  }, [trigger]);
+
+  const loginHandler = () => {
+    if (userDetails.password.match(validation)) {
+      login(userDetails);
+    } else if (!userDetails.email || !userDetails.password) {
+      setError({
+        text: "Dont leave any field empty",
+        state: true,
```

Comment:
suggestion: if its always true, can that be set by default

Reply:
thanks for pointing it out